app/controllers/index.js

```diff
@@ -0,0 +1,32 @@
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import ENV from 'website-my/config/environment';
+
+const BASE_URL = ENV.BASE_API_URL;
+
+export default class IndexController extends Controller {
+  @tracked status = this.model;
+
+  @action async updateStatus(status) {
+    this.status = status;
+    try {
+      const response = await fetch(`${BASE_URL}/users/self`, {
+        method: 'PATCH',
+        body: JSON.stringify({
+          status,
+        }),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'include',
+      });
+      const { status } = response;
+      if (status !== 204) {
```

Comment:
Maybe check for `response.ok` or something? I don't exactly remember but can you check the response object of `fetch` API and see if we can use something else than directly using the status code here? Could be changed to 200 tomorrow and our code will break in that case

Reply:
We can setup constants for status codes manually and import them.

Reply:
Yess it is `response.ok`. It is for the range of 200-299

Reply:
Cool, let's use that then

Reply:
@swarajpure If I am using `response.ok` then if the response is between 200-299, it will show the alert every time and here, we have to be specific to 204, so in this case, we should make a constant and use it throughout the repo, no other option is there for specific codes.

Reply:
How will it show alert every time? 🤔

Reply:
`response.ok` returns boolean

Reply:
ohh ok sorry, I was checking it the wrong way.

Reply:
Not using it? 🤔

Reply:
```
if (!response.ok) {
        alert('Something went wrong!');
}
```
This is working fine. Using it this way.